plugins/metadata/lib/kMetadataManager.php

```diff
@@ -479,7 +479,7 @@ public static function diffMetadataProfile(MetadataProfile $metadataProfile, $pr
 	 *
 	 * returns bool
 	 */
-	public static function validateMetadata($metadataProfileId, $metadata, &$errorMessage, $compareAgainstPreviousVersion = false)
```

Comment:
This is a  function definition and not call time, same as validateMetadataObjects.